packages/sources/taapi/src/adapter.ts

```diff
@@ -10,13 +11,14 @@ const customParams = {
   interval: true,
 }
 
-export const execute: Execute = async (input) => {
-  const validator = new Validator(input, customParams)
+// TODO: Run tests with valid pro tier + API Key
+export const execute: ExecuteWithConfig<Config> = async (request, config) => {
+  const validator = new Validator(request, customParams)
   if (validator.error) throw validator.error
 
   const jobRunID = validator.validated.id
   const indicator = validator.validated.data.indicator
-  const url = `https://api.taapi.io/${indicator}`
+  const url = `${config.api.baseURL}${indicator}`
```

Comment:
```suggestion
  const url = indicator
```

BaseURL will be handled if you spread `config.api` into line 36.